Dockerfile_test

```diff
@@ -0,0 +1,13 @@
+FROM hmctspublic.azurecr.io/base/java:openjdk-11-debug-distroless-1.2
```

Comment:
I assume this is just for testing? or do you think we will need one for each app, the hope was that we wouldn't have to add another dockerfile to each repo